### PR TITLE
Refactor, and fix, _compose_ssds_key

### PR DESCRIPTION
**Pull request by xbrianh**
URL length check previously omitted a "/"

### CODE REVIEW

File: ssds/__init__.py [hunk 1]
```diff
@@ -118,10 +118,8 @@ def _check_name_exists(self, submission_id: str, name: Optional[str]) -> str:
         return name
 
     def _compose_ssds_key(self, submission_id: str, submission_name: str, path: str) -> str:
-        path = path.strip("/")
-        dst_prefix = f"{submission_id}{self._name_delimeter}{submission_name}"
-        ssds_key = f"{dst_prefix}/{path}"
-        blobstore_key = f"{self.prefix}{ssds_key}"
+        ssds_key = f"{submission_id}{self._name_delimeter}{submission_name}/{path.strip('/')}"
```
Comment (DailyDreaming) [inline ssds/__init__.py:121]: Might be good to have a docstring here about the key naming conventions used here, or if this information is elsewhere, a link to the documentation.

**Review by DailyDreaming — APPROVED** [review]
LGTM.  See comment.

File: ssds/__init__.py (hunk 1 above)
Comment (DailyDreaming) [reply ssds/__init__.py:121]: Also, `delimeter` -> `delimiter`

Comment (xbrianh) [reply ssds/__init__.py:121]: Already documented in readme.

spelling fixed.

Comment (xbrianh) [reply ssds/__init__.py:121]: Actually, I'll need to fix the spelling error in a future PR

**Pull request merged**